iceoryx_posh/source/capro/service_description.cpp

```diff
@@ -29,11 +29,11 @@ ServiceDescription::ClassHash::ClassHash() noexcept
 
 ServiceDescription::ClassHash::ClassHash(const std::initializer_list<uint32_t>& values) noexcept
 {
-    uint64_t index = 0u;
+    uint64_t index = 0U;
     for (auto& v : values)
     {
         data[index++] = v;
-        if (index == 4u)
+        if (index == 4U)
```

Comment:
You touched the fishy! :fish: :tropical_fish: 

Why not use `CLASS_HASH_ELEMENT_COUNT` instead of this magic number `4U`.